### ISSUE TITLE
__outlined layers are broken in PRISM

### ISSUE THREAD

**Issue by Schpidi** [EOxServer/eoxserver#402]

**Comment by constantinius** [comment]
```xml
<?xml version='1.0' encoding='iso-8859-1'?>
<ogc:ServiceExceptionReport xmlns:ogc="http://www.opengis.net/ogc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.3.0" xsi:schemaLocation="http://www.opengis.net/ogc http://schemas.opengis.net/wms/1.3.0/exceptions_1_3_0.xsd">
  <ogc:ServiceException code="AttributeError">'OutlinedBrowseLayer' object has no attribute 'range'</ogc:ServiceException>
</ogc:ServiceExceptionReport>
```

**Comment by constantinius** [comment]
Fix in 5a1745f10262513741c6f00dbd5376f47eae249c
Needs testing
